examples/http-ts/src/queryToStore.ts

```diff
@@ -0,0 +1,25 @@
+import { URLSearchParams } from 'url';
+import * as store from '@leapfrogtechnology/async-store';
+
+/**
+ * Get the query params from the url and set them to async-store with `query` key.
+ *
+ * @param {URLSearchParams} params
+ */
+export default function queryToStore(params: URLSearchParams) {
```

Comment:
Instead of reinventing the wheel - use a simple query string parsing utility - https://www.npmjs.com/package/qs.